lib/rouge/lexers/ceylon.rb

```diff
@@ -51,7 +51,7 @@ class Ceylon < RegexLexer
 
         rule %r("(\\\\|\\"|[^"])*"), Literal::String
         rule %r('\\.'|'[^\\]'|'\\\{#[0-9a-fA-F]{4}\}'), Literal::String::Char
-        rule %r(".*``.*``.*"', String::Interpol
+        rule %r("[^`]*``[^`]*``[^`]*"', String::Interpol
         rule %r(\.)([a-z_]\w*)) do
```

Comment:
Usually for interpolation it's best to try and push a separate state, so you can highlight the contents of the interpolation as well. It can check for the closing interpolation tokens and then mixin the main expression state.

Reply:
Thanks @jneen. Great point. In fact, I have a follow-up commit locally to move this string interpolation to `state` (similar to how the `ruby` lexer deal with this). Once this merge , I can create another PR to address interpolation more appropriately.

Reply:
@tancnle I've fixed the wrapping as suggested in the latest push.